pkg/tools/fit.go

```diff
@@ -0,0 +1,25 @@
+package tools
+
+import (
+	"fmt"
+
+	"github.com/9elements/converged-security-suite/v2/pkg/intel/metadata/fit"
+)
+
+// PrintFit prints the Firmware Interface Table in a tabular human readable form.
+func PrintFit(image []byte) error {
```

Comment:
According to https://github.com/golang/go/wiki/CodeReviewComments#initialisms it should be `PrintFIT` :)

Reply:
Thanks for the review :) fixed it